README.md

```diff
@@ -81,8 +81,8 @@ most complete trading signals across a variety of asset classes.
 # Features
 
 🛠️&nbsp;Open Source Strategy Building Techniques (In Our Taoshi Community)<br>
-🫰&nbsp;Signals From a Variety of Asset Classes - Forex, Indices, Crypto<br>
-📈&nbsp;Higher Payouts to Top Traders<br>
+🫰&nbsp;Signals From a <a href="https://github.com/taoshidev/proprietary-trading-network/blob/main/vali_config.py#L161"> Variety of Asset Classes</a> - Forex, Indices, Crypto<br>
```

Comment:
might be helpful to mention that we're excluding GDAXI and FTSE somewhere

Reply:
Good catch just added a comment